test/e2e_flags.go

```diff
@@ -89,5 +92,27 @@ func initializeServingFlags() *ServingEnvironmentFlags {
 		"Set this flag to the MB of memory consumed by your service in resource limit tests. "+
 			"You service is set with 300 MB memory limit and shoud return a non-200 response when consuming such amount of memory.")
 
+	flag.StringVar(&f.RequestHeaders, "request-headers", "",
```

Comment:
Should we distinguish better that these requests headers are sent to the deployed KServices and not the control/API server?

Reply:
Do you have any suggestions for the name?

Reply:
I can't think of anything that isn't super long :(